src/uplift/tests.py

```diff
@@ -1,16 +1,14 @@
 from os import environ
 
-from models import Request, Test, TestGroup
+from models import Pod, Request, Test, TestGroup
 
 BASE_URL = environ["UPLIFT_BACKEND_URL"]
 
 tests = [
     Test(
         name="Gyms on Campus query",
-        request=Request(
-            method="GET", url=BASE_URL + "?query=query%7B%0Agyms%7B%0Aname%0A%7D%0A%7D"
```

Comment:
could `?query=query%7B%0Agyms%7B%0Aname%0A%7D%0A%7D` be put in a constant somewhere?